tests/script_annotations_outputs/script_annotations_output_param_and_artifact_mix.py

```diff
@@ -0,0 +1,29 @@
+try:
```

Comment:
For my own understanding - why are these tests vs examples? What are these testing?

Reply:
Oh, nvm, I looked up the name of the files and these files are actually used by the table style tests that follow! Maybe a small note at the top of each module might be helpful to clarify how these are tested?

Reply:
I added a line to all the test files to explain what the purpose is